### ISSUE TITLE
Buggy needs a fueling pad to be placed on

### ISSUE THREAD

**Issue by EzerArch** [micdoodle8/Galacticraft#3888]
Not a bug but not a suggestion either. It's an UIX flaw.

I attempt several time to place the buggy on the ground, and nothing. Then... "what if I place it on the fueling pad first", aha!

Please add a prompt like "The buggy needs a fueling pad to be placed on" or something.

Unless, it is a bug. XD

**Comment by radfast** [comment]
Ez, it's the same mechanic as a rocket.  You can't place a rocket on bare ground.

We should add a prompt, to help players.  Maybe one for the rockets as well.  They will need to be translated ...

**Comment by EzerArch** [comment]
> You can't place a rocket on bare ground.

Yes... but... it's expectable that a car-type vehicle is placed on ground. You don't expect a car would need a launcher or pad of some sort (unless you have a very fancy car).
